packages/plugins/Pins/README.md

```diff
@@ -14,22 +14,30 @@ The usage of `displayComponent` has no influence on the creation of Pins on the
-| toastAction      | string?       | If `boundaryLayerId` is set, and the pin is moved or created outside the boundary, this string will be used as action to send a toast information to the user. If no toast information is desired, leave this field undefined; for testing purposes, you can still find information in the console. |
+| fieldName        | type           | description                                                                                                                                                                                                                                                                                         |
+|------------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| toZoomLevel      | number         | Zoom level to use on outside input by e.g. address search.                                                                                                                                                                                                                                          |
+| appearOnClick    | appearOnClick? | Pin restrictions. If not set, a pin can be set on all zoom levels and will be visible.                                                                                                                                                                                                              |
+| boundaryLayerId  | string?        | Id of a vector layer to restrict pins to. When pins are moved or created outside of the boundary, an information will be shown and the pin is reset to its previous state. The map will wait at most 10s for the layer to load; should it not happen, the geolocation feature is turned off.        |
+| coordinateSource | string?        | The pins plugin may react to changes in other plugins. This specifies the path to such store positions. The position must, when subscribed to, return a GeoJSON feature.                                                                                                                            |
+| initial          | number[]?      | Configuration options for setting an initial pin.                                                                                                                                                                                                                                                   |
+| movable          | boolean?       | Whether user may drag pin with mouse. Defaults to false.                                                                                                                                                                                                                                            |
```

Comment:
While this is technically correct (and, therefore, the best kind of correct), we also need to implement that the cursor resists being moved by *clicking* another position. Currently, it can be moved by clicking.

Reply:
I'm thinking about changing `movable` to accept the strings `drag`, `click` and `none`, where

- `drag` accepts dragging and clicking
- `click` only accepts clicking
- `none` won't budge

when trying to move the pin. `movable` would still accept a boolean for now, but that configuration would become deprecated. Would that suffice?

Reply:
Sounds great. 👍

Reply:
Done f7f5fa6f